go/vt/vtgate/tabletgateway_flaky_test.go

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
Is this test still flaky? Or should these tests be merged into `tabletgateway_test.go`?

Reply:
this looks new tests to me, why will this be flaky?

Reply:
There is a fifteen second timeout in the end, it hasn't failed on my machine until now, but I am not certain whether it will work perfectly on the CI